eslint/eslint-bulk/src/start.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { execSync } from 'child_process';
+import * as process from 'process';
+import * as fs from 'fs';
+import * as path from 'path';
+
+interface IEslintBulkConfigurationJson {
+  /**
+   * `@rushtack/eslint`-bulk should report an error if its package.json is older than this number
+   */
+  minCliVersion: string;
+  /**
+   * `@rushtack/eslint-bulk` will invoke this entry point
+   */
+  cliEntryPoint: string;
+}
+
+function findPatchPath(): string {
+  let eslintrcPath: string;
+  if (fs.existsSync(path.join(process.cwd(), '.eslintrc.js'))) {
+    eslintrcPath = '.eslintrc.js';
+  } else if (fs.existsSync(path.join(process.cwd(), '.eslintrc.cjs'))) {
+    eslintrcPath = '.eslintrc.cjs';
+  } else {
+    console.error(
+      '@rushstack/eslint-bulk: Please run this command from the directory that contains .eslintrc.js or .eslintrc.cjs'
+    );
+    process.exit(1);
+  }
+
+  const env: { [key: string]: string } = { ...process.env, _RUSHSTACK_ESLINT_BULK_DETECT: 'true' };
+
+  let stdout: Buffer;
+  try {
+    stdout = execSync(`echo "" | eslint --stdin --config ${eslintrcPath}`, { env, stdio: 'pipe' });
```

Comment:
You can pass the value of stdin using `execSync`'s `input` option, which is a lot safer and more portable than using a shell command and calling `echo`: https://nodejs.org/docs/latest-v18.x/api/child_process.html#child_processexecsynccommand-options

Reply:
Works perfect, thanks David!